cypress/integration/score_card_spec.js

```diff
@@ -18,6 +18,53 @@ describe('Scorecard user flow', () => {
       .get('p').contains('2')
       .get('.plus-btn').eq(1).click()
       .get('p').contains('4')
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.next-hole-btn').click()
+      .get('.minus-btn').eq(0).click()
+      .get('.plus-btn').eq(1).click()
+      .get('.end-round-btn').click()
+
+    cy.get('.round-par')
+    cy.get('.round-date')
+    cy.get('h2').contains('Johnson & Wales University')
+    cy.get('h3').eq(0).contains('Phil')
+    cy.get('.player-total').eq(0).contains('14')
+    cy.get('.player-score').eq(0).contains('-25')
+    cy.get('h3').eq(1).contains('Evan')
+    cy.get('.player-total').eq(1).contains('130')
+    cy.get('.player-score').eq(1).contains('91')
```

Comment:
This test goes through the entire process of creating a round, playing a round, and then viewing the scores. I wonder if we should split this up somehow.